Json.More/JsonSerializerOptionsExtensions.cs

```diff
@@ -59,23 +59,6 @@ public static void Write<T>(this JsonSerializerOptions options, Utf8JsonWriter w
 	{
 #pragma warning disable IL2026, IL3050 // This helper is expected to be called with an options object that covers the needed TypeInfos.
 		JsonSerializer.Serialize(writer, value, options);
-#pragma warning restore IL2026, IL3050
-	}
-
-	/// <summary>
-	/// Write an object to JSON. If the type is known, prefer Write<![CDATA[<T>]]>
-	/// </summary>
-	/// <remarks>
-	/// A converter may throw any Exception, but should throw <cref>JsonException</cref> when the JSON is invalid.
-	/// </remarks>
-	/// <param name="options">The <see cref="JsonSerializerOptions"/> being used.</param>
-	/// <param name="writer">The <see cref="Utf8JsonReader"/> to read from.</param>
-	/// <param name="value">The value to serialize.</param>
-	/// <returns>The value that was converted.</returns>
-	public static void Write(this JsonSerializerOptions options, Utf8JsonWriter writer, object? value)
```

Comment:
Was this not useful?  Do you think it could be useful to users who aren't using AOT?

Reply:
It was no longer called. We could include it with an explicit type parameter. The problem I had was that it broke one of the tests so I updated the caller to go back to Serialize. But as I'm typing this comment I realize it's probably that we didn't pass the explicit type. I can fix that up.

Reply:
Don't worry about it.